### PR TITLE
publish new real-time event when upserted sample has not changed

### PR DESCRIPTION
**Pull request by iamigo**
Waiting for https://github.com/salesforce/refocus/pull/947 to be merged first

### CODE REVIEW

**Comment by harshkothari410** [comment]
Looks good to me.

File: realtime/redisPublisher.js [hunk 1]
```diff
@@ -134,6 +141,39 @@ function publishSample(sampleInst, subjectModel, event, aspectModel) {
...
+    status: sample.status, // for socket.io perspective filtering
+    absolutePath: sample.absolutePath, // used for persp filtering
+    updatedAt: sample.updatedAt,
+    subject: {
+      absolutePath: sample.absolutePath,
+      tags: sample.subjectTags, // for socket.io perspective filtering
+    },
+    aspect: {
+      name: sample.aspectName, // for socket.io perspective filtering
+      tags: sample.aspectTags, // for socket.io perspective filtering
```
Comment (jgraff2) [inline realtime/redisPublisher.js]: Should we maybe strip out these fields in the emitter or perspective so it's clear to the lens that it shouldn't rely on this object for any data other than updatedAt?

Comment (iamigo) [reply realtime/redisPublisher.js]: I like that idea--let me take a stab at that!!!

The first benefit is as you wrote, but another benefit is smaller event data means less work serializing, and as we saw today, that takes up a lot of our time!

**Comment by iamigo** [comment]
updated the pr per Jon's comment, will test in a heroku environment on monday

**Comment by iamigo** [comment]
@jgraff2 @harshkothari410 @pallavi2209 -- updated after jon's comment, tested in devbox, ready for one more round of review before merging

**Pull request merged**